go.mod

```diff
@@ -0,0 +1,32 @@
+module terraform-provider-aap
```

Comment:
I don't know why the TF docs tell you to do this, but we should use `github.com/ansible/terraform-provider-app` for the module name. It's probably unlikely this would be used as a dependency, but all of go mod's tooling is built around the idea that dependencies are downloadable.

Reply:
Updated